core/src/main/java/org/apache/carbondata/core/metadata/schema/table/column/ColumnSchema.java

```diff
@@ -131,6 +131,11 @@
    */
   private String timeSeriesFunction = "";
 
+  /**
+   * set whether the column enables page bloom or not.
+   */
+  private boolean isPageBloomColumn = false;
```

Comment:
need to handle in `readFields `and `writeFields `method for seralization and deseralization